src/plugins/janus_videoroom.c

```diff
@@ -8617,6 +8620,11 @@ static void *janus_videoroom_handler(void *data) {
 								desc_updated = TRUE;
 								g_free(ps->description);
 								ps->description = g_strdup(d_desc);
+								if(ps->rc) {
+									if(ps->rc->description)
+										g_free(ps->rc->description);
+									ps->rc->description = g_strdup(d_desc);
+								}
```

Comment:
See above: just use a setter and you don't have to worry about freeing and duping. It's important to point out that this call would be useless when invoked after the mjr header has been written and the recorder is working, since we only write it down before all that happens.

Reply:
I turned this into a setter. Now that you mention it, I think this can be omitted. Since the value is copied when the recorder is instantiated, and that's enough, I believe?

Reply:
I removed this part, and it (still) works as expected.